rust-toolchain.toml

```diff
@@ -2,5 +2,5 @@
 # SPDX-License-Identifier: Apache-2.0 OR MIT
 
 [toolchain]
-channel = "nightly-2022-10-11"
-components = ["llvm-tools-preview", "rustc-dev", "rust-src", "rustfmt"]
+channel = "nightly-2022-10-25"
+components = ["llvm-tools-preview", "rustc-dev", "rust-src", "rustfmt"]
```

Comment:
nit: your editor might not be configured to emit trailing newlines

for vscode it looks like I added `"files.insertFinalNewline": true,` to its "user settings"

Reply:
thanks - i did change that.